Formula/pumba.rb

```diff
@@ -24,5 +24,7 @@ def install
   test do
     output = shell_output("#{bin}/pumba --version")
     assert_match "Pumba version 0.4.4", output
+    output = pipe_output("#{bin}/pumba rm test-container 2>&1")
```

Comment:
0 is the default asserted by shell_output, but it's kind of surprising for the exit code to be 0 if the command failed. Why is that?